.meteor/versions

```diff
@@ -1,10 +1,10 @@
 abernix:minifier-js@1.3.19
 abernix:standard-minifier-js@1.3.19
-accounts-base@1.2.16
+accounts-base@1.2.17
 accounts-facebook@1.1.1
 accounts-google@1.1.2
 accounts-oauth@1.1.15
-accounts-password@1.3.5
+accounts-password@1.3.6
```

Comment:
Please revert these version numbers back if they did not affect what you did, changing them usually break the Heroku deployment.